sdk/rpc/server/server.go

```diff
@@ -136,17 +137,18 @@ func (t *RPCServer) CredentialProvisionerDescription(req proto.ProvisionerID, re
 // interface. The call is forwarded to the Provision() function of the Provisioner of the credential identified by
 // req.CredentialID.
 func (t *RPCServer) CredentialProvisionerProvision(req proto.ProvisionCredentialRequest, resp *sdk.ProvisionOutput) error {
+	defer func() {
+		if err := recover(); err != nil {
+			resp.AddError(fmt.Errorf("your locally built plugin failed with the following panic: %s; and with stack trace: %s", err, string(debug.Stack())))
```

Comment:
```suggestion
			resp.AddError(fmt.Errorf("your locally-built plugin panicked with the following error: %s; and with stack trace: %s", err, string(debug.Stack())))
```

Reply:
Thanks for the suggestion! I will apply this in the separate MR made especially for this panic catching.